src/content/docs/rules.mdx

```diff
@@ -56,6 +57,15 @@ title: Rules
             1. The exchange rate for the coins to ingame currency may be changed, but within reason and non frequent.
         4. If there isn't an exchange between non currency used while gambling and ingame currency, this should be properly communicated to players on an ingame monitor, poster or sign and in the user interface where the player gambles.
         5. Any currency gained while gambling must be able to be paid out by the gambling operator to the player at all times.
+    14. Exploiting ingame player software over modem or physical interaction (monitor clicks / terminal access / disk drives) is allowed. Player software, config or data should *not* be removed outright.
+        1. This can be for the goal of software piracy, a proof of concept, gaining an advantage in a game or adding "points".
+            1. When changing any value in a configuration or database, it should only be your own player data that is affected.
+            2. When in doubt of if a reason to exploit is allowed, ask staff *before* exploiting.
+        2. Player software, databases, configuration or other files on a computer should not be removed outright.
```

Comment:
Pretty sure this is already established in the main point.

Also the use of the word "outright" is not good here